src/components/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { Component, MouseEvent } from 'react';
+
+export type tabType = 'basic';
+
+interface IProps {
+  headings: string[];
+  tabType?: tabType;
+  className?: string;
+  content: string[];
+}
+interface IState {
+  activeHeading: string;
+  description: string;
+}
+
+class Tabs extends Component<IProps, IState> {
+  static defaultProps = {
+    tabType: 'basic',
+  };
+  public constructor(props: IProps) {
+    super(props);
+
+    this.state = {
+      activeHeading:
+        this.props.headings && this.props.headings.length > 0
+          ? this.props.headings[0]
+          : '',
+      description: this.props.content[0],
```

Comment:
why this is called content in the props, but the description in the state.
For the real-world situation, the content should not just be a string. it could be any JSX.

Reply:
My logic is to set a state as description and then set description to the pointed string in the content. I just don't know what is the way to set the content to be any JSX

Reply:
check this: https://www.digitalocean.com/community/tutorials/react-tabs-component

Reply:
Hi Patrick, I just pushed the edited code. Could you please check for it?